### DISCUSSION TITLE
keycloak console browser compability

### DISCUSSION THREAD

**Discussion by kingyue737** [keycloak/keycloak#19712] [Q&A] [answered]
I want to implement keycloak sso for our clients. Since their browsers are installed in intranet and pinned at an old version (like Chrome <= 70), I wonder whether admin console of keycloak supports old browsers. Is there a list showing browsers supported by each keycloak version?

**Comment by jonkoops** [comment, marked as answer]
We don't have an official browser support target AFAIK, but unofficially we try to keep an moving support target. Supported browsers include:

- The latest two major versions of 'evergreen' browsers (Chrome, Firefox).
- The latest two minor versions of Safari (e.g 16.3, 16.4).
- The versions of Firefox [Extended Support Release](https://www.mozilla.org/en-US/firefox/enterprise/) (ESR) actively supported by Mozilla.

Any older versions we do not actively support, unless it would cause a huge fallout in the larger community somehow.

As for your intranet, what your organization is doing is bad practice and insecure. Those versions of Chrome are full of security holes that could be actively exploited by a malicious actor that gets access to the network, including Arbitrary Code Execution.
